### PR TITLE
Change design of notification about lost connections in web UI

### PR DESCRIPTION
**Pull request by Gargron**
grafik

### CODE REVIEW

**Comment by digitalcircuit** [comment]
The newer message seems more clear and easier to understand!

That said, I am unsure if `An admin` could be confusing since moderators can block/suspend, too.  Though I get that "a moderator" doesn't shorten down as nicely ("a mod" could sound like a browser/etc modification), and this distinction might not matter in practice.

(Regardless, even as someone running a single-user instance, I am immensely happy to see the overall feature making it into v4.3.0.)

**Review by ClearlyClaire — COMMENTED** [review]
The wording is an improvement, and so is the overall design, but it does not really fit with our other notification designs, because of the much larger padding, the larger icon, and the fact that everything is padded right:
image

Shouldn't it use the same padding and icon size as the other notifications (while possibly changing all of them in a future PR)?
image

**Comment by Gargron** [comment]
I think that the uniqueness of this notification makes it possible to have it use the new design first. As a second step I would update all other non-mention notifications to have the new design.

**Review by ClearlyClaire — APPROVED** [review]
I don't understand the insistence for introducing the design inconsistency, but it's not a big deal. Looks good to me otherwise, and the wording is a clear improvement.

File: app/javascript/mastodon/features/notifications/components/relationships_severance_event.jsx [hunk 1]
```diff
@@ -2,60 +2,44 @@ import PropTypes from 'prop-types';
...
-import { RelativeTimestamp } from 'mastodon/components/relative_timestamp';
-
-// This needs to be kept in sync with app/models/relationship_severance_event.rb
+// This needs to be kept in sync with app/models/relationships_severance_event.rb
 const messages = defineMessages({
-  account_suspension: { id: 'relationship_severance_notification.types.account_suspension', defaultMessage: 'Account has been suspended' },
-  domain_block: { id: 'relationship_severance_notification.types.domain_block', defaultMessage: 'Domain has been suspended' },
-  user_domain_block: { id: 'relationship_severance_notification.types.user_domain_block', defaultMessage: 'You blocked this domain' },
+  account_suspension: { id: 'notification.relationships_severance_event.account_suspension', defaultMessage: 'An admin from {from} has suspended {target}, which means you can no longer receive updates from them or interact with them.' },
+  domain_block: { id: 'notification.relationships_severance_event.domain_block', defaultMessage: 'An admin from {from} has blocked {target}, including {followersCount} of your followers and {followingCount, plural, one {# account} other {# accounts}} you follow.' },
```
Comment (CyberTailor) [inline app/javascript/mastodon/features/notifications/components/relationships_severance_event.jsx:12]: `followersCount` also needs plural forms

**Pull request merged**